dist/ng-obiba-mica.js

```diff
@@ -9951,28 +9952,30 @@ ngObibaMica.lists
         templateUrl: 'lists/views/input-search-widget/suggestion-field.html',
         link: function (scope) {
           scope.suggest = function (query) {
-            if (scope.documentType && query && query.length > 1) {
-              var rql = RqlQueryService.parseQuery($location.search().query);
-              var targetQuery = RqlQueryService.findTargetQuery(typeToTarget(scope.documentType), rql);
-              var classNameQuery = RqlQueryService.findQueryInTargetByVocabulary(targetQuery, 'className');
-              if (classNameQuery) {
-                query = 'className:' + classNameQuery.args[1] + ' AND (' + query.replace(/\/.*/, '') + ')';
-              }
-
-              return DocumentSuggestionResource.query({locale: $translate.use(), documentType: scope.documentType, query: query})
-                  .$promise
-                  .then(function (response) {
-                      var parsedResponse = Array.isArray(response) ? response : [];
-
-                      for (var i = 0; i < parsedResponse.length; i++) {
-                        parsedResponse[i] = parsedResponse[i].replace(/\/.*/, '');
-                      }
-
-                      return parsedResponse;
-                  });
-            } else {
-              return [];
-            }
+            return EntitySuggestionService.suggest(scope.documentType, query);
+
+            // if (scope.documentType && query && query.length > 1) {
```

Comment:
What's the purpose of keeping commented code?

Reply:
Fixing it now